R/page.R

```diff
@@ -130,6 +122,23 @@ page_navbar <- function(..., title = NULL, id = NULL, selected = NULL,
   )
 }
 
+# https://github.com/rstudio/shiny/issues/2310
+get_window_title <- function(title = NULL, window_title = NA) {
```

Comment:
Very much a nit, but I'd prefer another verb besides `get_`, which to me connotes the involvement of an external resource or a global or session option.

I didn't have any strong candidates in mind until just now when I noticed that you used the verb _infer_ in the warning.

```suggestion
infer_window_title <- function(title = NULL, window_title = NA) {
```

Reply:
Ah yes, I like this better, will adopt this naming in #588